registration/include/pcl/registration/sample_consensus_prerejective.h

```diff
@@ -285,8 +301,21 @@ namespace pcl
         * \param inliers indices of source point cloud inliers
         * \param fitness_score output fitness score as RMSE 
         */
+      PCL_DEPRECATED(1, 15, "Please use `getFitness(final_transformation_, inliers)` instead")
       void 
-      getFitness (std::vector<int>& inliers, float& fitness_score);
+      getFitness (std::vector<int>& inliers, float& fitness_score) const;
+
+      /** \brief Obtain the fitness of a transformation
+        * The following metrics are calculated, based on
+        * \b final_transformation and \b corr_dist_threshold_:
+        *   - Inliers: the number of transformed points which are closer than threshold to NN
+        *   - Error score: the MSE of the inliers  
+        * \param transformation transformation to be evaluated
+        * \param inliers indices of source point cloud inliers
+        * \return fitness_score fitness score as RMSE
```

Comment:
So, three lines above it says the error is `MSE of the inliers` and here it says `RMSE`. What's actually calculated, though, is the mean nearest neighbor distance.

Reply:
Hmmm, it looks ```getFitness()``` calculates the mean of the square distances of inliers, and MSE is correct. I updated the doc as well as the original one just above it.